src/components/DevNavMenu.tsx

```diff
@@ -685,22 +685,26 @@ DevNavMenuTuts.set('inkWorkshop', [
     ],
   },
 ])
-DevNavMenuTuts.set('cumulusWorkshop', [
+DevNavMenuTuts.set('cumulusTutorial', [
   {
-    name: `Substrate Cumulus Workshop`,
+    name: `Cumulus Tutorial`,
     items: [
       {
-        title: `Getting Started`,
-        link: '/tutorials/v3/cumulus-workshop/pt1',
+        title: `Start a relay chain`,
+        link: '/tutorials/v3/cumulus/pt1',
       },
       {
-        title: `Connecting to a Relay Chain`,
-        link: '/tutorials/v3/cumulus-workshop/pt2',
+        title: `Connect a parachain`,
+        link: '/tutorials/v3/cumulus/pt2',
       },
       {
-        title: `Launch a Development Environment`,
-        link: '/tutorials/v3/cumulus-workshop/pt3',
+        title: `Launch a parachain testnet`,
+        link: '/tutorials/v3/cumulus/pt3',
       },
+      //   {
+      //     title: `Register on Rococo`,
+      //     link: '/tutorials/v3/cumulus/pt4',
+      //   },
```

Comment:
I explicitly want to build this page, but don't want it in the nav bar (yet) as rococo isn't accepting community registrations right now.